content/pages/disability-benefits/claims-appeal.md

```diff
@@ -21,7 +21,7 @@ When the Regional Office (RO) receives your Notice of Disagreement (NOD), you'll
 <a class="usa-button-primary" href="/track-claims">Track your appeal</a>
 </div>
 
-### Appeals Process Overview
+<h3 class="no-margin">Appeals Process Overview</h3>
```

Comment:
Unless there's a specific style here that's overriding it, I think `h3` already has `margin: 0` set in `_va.scss`.